docs/layers/ecs/tutorials/1password-scim-bridge.mdx

```diff
@@ -0,0 +1,132 @@
+
+    <Steps>
+      1. Create a new stack configuration for the SCIM bridge. The placement of this file will depend on your project structure. For example, you could create a new file such as `stacks/catalog/ecs-services/1password-scim-bridge.yaml` with the following content:
+
+          <CollapsibleText type="medium">
+          ```yaml
+          import:
+            - catalog/terraform/services/defaults
+
+          components:
+            terraform:
+              1pass-scim:
+                metadata:
+                  component: ecs-service
+                  inherits:
+                    - ecs-service/defaults
+                vars:
+                  enabled: true
+                  name: 1pass-scim
+                  containers:
+                    service:
+                      name: op_scim_bridge
+                      image: 1password/scim:v2.9.5
+                      cpu: 128
+                      memory: 512
+                      essential: true
+                      dependsOn:
+                        - containerName: redis
+                          condition: START
+                      port_mappings:
+                        - containerPort: 3002
+                          hostPort: 3002
+                          protocol: tcp
+                      map_environment:
+                        OP_REDIS_URL: redis://localhost:6379
+                        OP_TLS_DOMAIN: ""
+                        OP_CONFIRMATION_INTERVAL: "300"
+                      map_secrets:
+                        OP_SESSION: "/path/to/secret" # TODO "${secret_arn}"
+                        # OP_WORKSPACE_CREDENTIALS: ""
+                        # OP_WORKSPACE_SETTINGS: ""
+                      log_configuration:
+                        logDriver: awslogs
+                        options: {}
+                    redis:
+                      name: redis
+                      image: redis:latest
+                      cpu: 128
+                      memory: 512
+                      essential: true
+                      restart: always
+                      port_mappings:
+                        - containerPort: 6379
+                          hostPort: 6379
+                          protocol: tcp
+                      map_environment:
+                        REDIS_ARGS: "--maxmemory 256mb --maxmemory-policy volatile-lru"
+                      log_configuration:
+                        logDriver: awslogs
+                        options: {}
+          ```
+          </CollapsibleText>
+      2. Confirm the `map_secrets` for `OP_WORKSPACE_CREDENTIALS` and `OP_WORKSPACE_SETTINGS` match the AWS SSM Parameter Store paths you created previously, and confirm they are in the same account and region as this ECS service component.
+      3. Deploy the ECS service with Atmos:
+        ```bash
+        atmos terraform apply 1pass-scim -s plat-usw2-dev
+        ```
```

Comment:
Similar to the above. I believe this is not sufficiently indented to be considered part of the previous bullet.

Reply:
fixed locally

![CleanShot 2024-08-20 at 13 01 07@2x](https://github.com/user-attachments/assets/6c65aef8-b598-4124-8bd8-1d9ba8b8b380)